packages/fluentui/react/src/themes/teams-high-contrast/componentStyles.ts

```diff
@@ -1,2 +1 @@
 export { default as MenuItem } from './components/Menu/menuItemStyles';
-export { default as Dropdown } from './components/Dropdown/dropdownStyles';
```

Comment:
Should be `dropdownStyles.ts` removed at all for High contrast?

Reply:
FYI: https://github.com/OfficeDev/office-ui-fabric-react/pull/12278 I merged these changes, so please rebase your branch 🙏

Reply:
I'll wait until the big comma/space change coming soon...